Doc/whatsnew/3.14.rst

```diff
@@ -514,6 +514,15 @@ sys
   from other interpreters than the one it's called in.
 
 
+tkinter
+-------
+
+* Add the argument ``**kw`` to method :meth:`!after` and method
```

Comment:
It is a var-keyword parameter, not argument (see https://docs.python.org/3/glossary.html#term-parameter).

I would write something like "The ... methods of ... now support passing keyword aguments to the function." In any case this needs a review by a native English speaker.

Reply:
Sorry, my translator isn't particularly accurate.